src/test/java/org/galatea/starter/entrypoint/IexRestControllerTest.java

```diff
@@ -0,0 +1,92 @@
+package org.galatea.starter.entrypoint;
+
+import static org.mockito.BDDMockito.given;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import junitparams.JUnitParamsRunner;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.galatea.starter.ASpringTest;
+import org.galatea.starter.domain.IexLastTradedPrice;
+import org.galatea.starter.domain.IexSymbol;
+import org.galatea.starter.service.IexService;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+
+@RequiredArgsConstructor
+@Slf4j
+// We don't load the entire spring application context for this test.
+@WebMvcTest(IexRestController.class)
+// Use this runner since we want to parameterize certain tests.
+// See runner's javadoc for more usage.
+@RunWith(JUnitParamsRunner.class)
+public class IexRestControllerTest extends ASpringTest {
+
+  @Autowired
+  private MockMvc mvc;
+
+  @MockBean
+  private IexService mockIexService;
+
+  @Test
+  public void testGetSymbolsEndpoint() throws Exception {
+
+    List<IexSymbol> mockResponse = new ArrayList<>();
```

Comment:
Another nit (given this is test code) but you could use:

List<IexSymbol> mockResponse = singletonList(IexSymbol.builder()...);